_data/projects/julia.yml

```diff
@@ -0,0 +1,9 @@
+name: Julia Language
+desc: Julia is a high-level, high-performance dynamic programming language for technical computing. https://www.julialang.org
```

Comment:
We don't render the links currently (see #312 for us supporting markdown) but using `https` here triggers a certificate error (as it expects julialang.org but gets github.com)